task/import.go

```diff
@@ -82,7 +86,7 @@ func TaskImport(tctx *TaskContext) (*data.TaskOutput, error) {
 		}
 		defer importedFile.Close()
 		// RecordStream on output file for HLS playback
-		playbackRecordingId, err = Prepare(tctx, metadata.AssetSpec, importedFile)
+		playbackRecordingId, err = Prepare(tctx, metadata.AssetSpec, importedFile, *fileInfoReader.Size, 50)
```

Comment:
`fileInfoReader` not always has a `Size`, which is why it is a pointer instead of a direct value.

One way would be to check before using, but I think it'll be much simpler to just grab the size from `metadata.AssetSpec` (so then you don't even need the separate arg, just grab it from inside `Prepare`)